src/azul/project/hca/__init__.py

```diff
@@ -20,7 +20,15 @@
                                   if config.dss_endpoint != "https://dss.data.humancellatlas.org/v1"
                                   else "files.biomaterial_json")
                     }
-                }
+                },
+                {
+                    "range": {
+                        "manifest.version": {
+                            "gte": "2018-10-10"
+                        }
+                    }
+                } if config.dss_endpoint == "https://dss.integration.data.humancellatlas.org/v1"
+                else {}
```

Comment:
Nope. See my previous comment. I don't want an empty dict in the else case. I want nothing. It was all there on a platter for you and you still ignored it: 

Use *([{"foo": bar}] if condition else [])